modules/afprog/afprog.c

```diff
@@ -358,18 +430,55 @@ afprogram_dd_init(LogPipe *s)
     }
   log_pipe_append(&self->super.super.super, (LogPipe *) self->writer);
 
-  return afprogram_dd_reopen(self);
+  return restore_successful ? TRUE : afprogram_dd_reopen(self);
+}
+
+static inline void
+afprogram_dd_store_reload_info(AFProgramDestDriver * const self, GlobalConfig * const cfg)
+{
```

Comment:
Since the compilers can prove the constness, this notation has effects only on humans.
For example, when I see this constant pointer, I won't expect to change its value, therefore I do not need to follow the changes of this variable.